backend/models/Adminuser.js

```diff
@@ -0,0 +1,18 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+var passportLocalMongoose = require('passport-local-mongoose');
+
+var AdminUser = new Schema({
+    
+    email:{
+        type:String,
+        default:"",
+    },
+    admin: {
+        type: Boolean,
+        default: true
```

Comment:
Here, the default value should be `false`. And there should be an API for updating this value which can only be called by the admins. So the flow would be like:

1. User signs up.
2. After successful signup, It will be shown on the admin panel(We would be implementing it very soon)
3. When any of the admins approve the user, the register user get the admin privileges.

`Just after signup, the user won't be considered as admin until some pre-existing admins verifies it.`